scripts/generate_trust_policy

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+
+# Helper script to generate trust policy needed to install operator using role-based authentication.
+# https://sagemaker.readthedocs.io/en/stable/amazon_sagemaker_operators_for_kubernetes.html#create-an-iam-role
+#
+# Run as:
+# $ ./generate_trust_policy ${EKS_CLUSTER_REGION} ${AWS_ACCOUNT_ID} ${OIDC_ID} > trust.json
+#
+# For example:
+# $ ./generate_trust_policy us-west-2 123456789012 D48675832CA65BD10A532F597OIDCID > trust.json
+# This will create a file `trust.json` containing a role policy that enables the operator in an EKS cluster to assume AWS roles.
+#
+
+cluster_region="$1"
+account_number="$2"
```

Comment:
We could also generate this from AWSCLI using `aws sts get-caller-identity | jq -r ".Account"`. Most people probably do not have their account number on hand, so this might otherwise require more documentation.

Reply:
I don't know.. while I agree that is useful, I think this script is more valuable as a dumb script. For example if someone has multiple AWS accounts and are setting it up for one but `aws` picks another. I think our users will be able to get their account number.

Reply:
Sorry I missed the overall comment on the point + justification.

I like your justification and think there's merit.. I'll make this script smarter so that it grabs everything.